iotdb/src/main/java/org/apache/iotdb/db/conf/directories/strategy/MaxDiskUsableSpaceFirstStrategy.java

```diff
@@ -35,7 +36,12 @@ public int getMaxSpaceFolder() throws DiskSpaceInsufficientException {
     long maxSpace = 0;
 
     for (int i = 0; i < folders.size(); i++) {
-      long space = getUsableSpace(folders.get(i));
+      String folder = folders.get(i);
+      if (!FileUtils.hasSpace(folder)) {
+        continue;
+      }
+
+      long space = FileUtils.getUsableSpace(folder);
       if (space > maxSpace) {
```

Comment:
should be "if( space >= maxSpace)... "
Suppose all folder are empty, if so, you will return -1...

Reply:
if all folders are empty, maxIndex will not be init as line 41 skip to next loop, so exception will be thrown.